client/lclusterc.go

```diff
@@ -0,0 +1,316 @@
+/*
+ * File: client/main.go
+ *
+ * Description: Contains the lcluster client code.
+ *
+ * TODO: removejob is broken, others funcs are unimplemented
+ */
+
+package main
```

Comment:
I have done a gofmt to get the code to style, as per the below commit:

https://github.com/rbisewski/lclusterd/pull/2/commits/3883575673a944e6ec334fb44bd79a89572b763f

Going to spend a bit more time examining the document to implement the remainder of the styles.